packages/docs/operations-manual/key-management/detailed.md

```diff
@@ -0,0 +1,148 @@
+
+Because of the significant priveleges afforded to the Locked Gold Account key, it is best to store this key securely and access it as infrequently as is possible. Authorizing other signer keys is one way to minimize how frequently you need to access your Locked Gold Account key. The Locked Gold Account key will only be used to send transactions and **can be stored on a Ledger hardware wallet.**
+
+### Creating a Locked Gold Account
+An undesignated key may be designated as a Locked Gold Account key by running the following command:
+
+```shell
+# Designate the key as a Locked Gold Account
+celocli account:create --from $ADDRESS_TO_DESIGNATE --useLedger
+
+# Confirm the key was designated as a Locked Gold Account
+celocli account:show $ADDRESS_TO_DESIGNATE
+```
+
+## Authorized Vote Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a vote signing key. Authorized vote signing keys can vote in validator elections and on-chain governance on behalf of the Locked Gold Account.
+
+Note that the vote signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
+
+Authorized vote signing keys will only be used to send transactions and **can be stored on a Ledger hardware wallet**.
+
+
+### Authorizing a Vote Signer
+An undesignated key may be authorized as a vote signer on behalf of a Locked Gold Account by running the following commands:
+
+```shell
+# Create a proof-of-possession. Note that the signer private key must be available.
+celocli account:proof-of-possession --account $LOCKED_GOLD_ACCOUNT --signer $KEY_TO_AUTHORIZE --useLedger
+
+# Authorize the vote signing key. Note that the Locked Gold Account private key must be available.
+celocli account:authorize --from $LOCKED_GOLD_ACCOUNT --role vote --signer $KEY_TO_AUTHORIZE --signature $PROOF_OF_POSSESSION --useLedger
+
+# Confirm that the vote signer was authorized
+celocli account:show $LOCKED_GOLD_ACCOUNT
+
+# You can also look up account info via the authorized signer
+celocli account:show $KEY_TO_AUTHORIZE
+```
+
+## Authorized Validator Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a validator signer. Authorized validator signing keys can be used to register and manage a validator or validator group on behalf of the Locked Gold Account. If the authorized validator key is used to register a validator it is also used to sign consensus messages.
+
+### Authorized Validator Signers for Validator Groups
+An authorized validator signer key that will be used to register a validator group can be used to send group management transactions (e.g. register, add member A, queue commission update to 0.25, etc.) Because this key does not participate directly in consensus it **can be stored on a Ledger hardware wallet.**
+
+### Authorized Validator Signers for Validators
```

Comment:
naive reader hat -- can I use the same validator signer for both group and validator?